README.md

```diff
@@ -1 +1,181 @@
+  - [Setup](#setup)
+  - [live-demo](#livelink)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Kanban Board Project App <a name="about-project"></a>
+
+>Kanban Board project is an API group project using M&N Resturant website API. We have created using HTML, CSS, Javascript, linters, and webpack.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1: Html
+2: CSS
+3: linters
+4: javascript
+5: webpack
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">Html</a></li>
+    <li><a href="https://w3schools.com/">CSS</a></li>
+    <li><a href="https://w3schools.com/">Javascript</a></li>
+    <li><a href="https://webpack.com/">webpack</a></li>
+  </ul>
+</details>
+</details>
+
+<!-- Features -->
+<p> the project explanation </p> 
+
+### Key Features <a name="key-features"></a>
+
+
+- Added HTML 
+- Added CSS 
+- Added javascript 
+- Added webpack as module bundler 
+- fetch data from real API 
+
+
+### Testing <a name="Testing"></a>
+
+- we used Jest as testing library in this project 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## live demo <a name="livelink">Coming Soon</a>
+- [clickhere](https://zuwairanajma.github.io/Javascript-Capstone_AlbertandJuw/dist)
+- [video](https://drive.google.com/file/d/1EyeCIsYWUFGKX8nFjz2ayp_4T8OqxJGL/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1: web browser
+2: a code editor
+3: node installed on your pc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd to your desired folder 
+  git clone https://github.com/Zuwairanajma/capstone-project.git
+### Install
+
+Install this project with:
+  install node_modules by this command : npm i -y
+
+then:
+
+run this command: npm run start 
+
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zuwaira Sadiq**
+- GitHub: [@Zuwairanajma](https://github.com/Zuwairanajma)
+- Twitter: [@SadiqJuwairiyya](https://twitter.com/SadiqJuwairiyya)
+- LinkedIn: [Zuwaira Sadiq](https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?)
+
+
+👤 **Albert Antwi**
+- GitHub: [@albertkantwi](https://github.com/albertkantwi)
+- Twitter: [@Albert_KAntwi](https://twitter.com/Albert_KAntwi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/albert-antwi-557291263/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- coming soon !!
```

Comment:
Kindly add 1-3 valid potential future features to be added to the project. This is a part of the general requirements.